xml/cap_depl_azure.xml

```diff
@@ -18,17 +18,24 @@
  
   <para>
    Starting with version 1.1, &productname; supports running on Microsoft Azure 
-   Container Service (AKS). This chapter describes how to prepare AKS for deploying
-   &productname;.
+   Container Service (ACS). This chapter describes how to prepare ACS for 
```

Comment:
`AKS` actually is correct. See: https://azure.microsoft.com/en-us/services/container-service/.

Reply:
Then this whole chapter needs to be rewritten, to take out the manual K8s setup  \o/

Reply:
Another issue is confusion between CAP and SCF-- we're deploying SCF on Azure Kubernetes, not the whole CAP stack.